fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1623,15 +1623,25 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 		                            idempotencyIdSet.param2 = kv.value;
 		                            auto& tags = pProxyCommitData->tagsForKey(kv.key);
 		                            self->toCommit.addTags(tags);
-		                            Arena arena;
-		                            writeMutation(self, INVALID_ENCRYPT_DOMAIN_ID, &idempotencyIdSet, nullptr, &arena);
+		                            if (self->pProxyCommitData->isEncryptionEnabled) {
```

Comment:
@sfc-gh-yiwu this place is wrapped in a function so i can't call an actor from it. So i wont call `writeMutation` here

Reply:
ah, I didn't realize `writeMutation` is now an actor.